modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -30,19 +32,29 @@ spec:
     node-role.kubernetes.io/worker: "" <3>
   desiredState:
     interfaces:
-      - name: br1
-        type: linux-bridge
-        state: absent <4>
+    - name: br1
+      type: linux-bridge
+      state: absent <4>
+    - name: eth1
+      type: ethernet <5>
+      state: up <6>
+      ipv4:
+        dhcp: true <7>
+        enabled: true <8>
 ----
 <1> Name of the policy.
 <2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
 <3> This example uses the `node-role.kubernetes.io/worker: ""` node selector to select all worker nodes in the cluster.
 <4> Changing the state to `absent` removes the interface.
+<5> The type of interface. This example creates an Ethernet networking interface.
+<6> The requested state for the interface after creation.
```

Comment:
Not "after creation". This interface already exists, and this entry defined the requested state after detaching it from the master interface (bridge, bond, ...)

Reply:
Updated to:
The requested state for the interface.